sigs/contributor-strategy.md

```diff
@@ -0,0 +1,173 @@
+# CNCF SIG Contributor Strategy Charter
+
+Primary Authors: Paris Pittman, Josh Berkus  
+
+Not approved // needs a TOC vote
+
+Reviewed and/or contributed to by:  
+* Matt Klein
+* Matt Farina  
+* Carolyn Van Slyck  
+* April Nassi
+* Matt Jarvis
+* Gerred Dillon
+* Ken Owens
+* Cheryl Hung
+* Amye Scavarda Perrin
+* Ihor Dvoretskyi
+
+
+## Introduction
+This charter describes the operations of the CNCF Special Interest Group (SIG)
+Contributor Strategy. This SIG is responsible for contributor experience,
+sustainability, governance, and openness guidance to help CNCF community groups
+and projects with their own contributor strategies for a healthy project.
+
+Our initial three stakeholders:  
+1 - CNCF projects and their contributors/maintainers,  
+2 - End Users in the broader community and member companies,  
+3 - TOC
+
+## Mission
+Consistent with the CNCF SIG definition, the mission of CNCF SIG Contributor
+Strategy is to collaborate on strategies related to building, scaling, and
```

Comment:
I love this initiative!  Just a bit on wording... I'd like to see a mission that is a bit more active, perhaps tied to CNCF mission. What does success look like?  I think this effort will drive a healthy ecosystem and increase growth/retention. Of course, it will require participation of the projects and if the SIG is doing all the right things (which from everything I hear you are on track to do) then projects will be eager to participate / take advantage of resources provided by this SIG.

Reply:
Any suggestions on wording?  You seem to have grasped what we'll be doing, what would be a better way to phrase this?

Reply:
I added a suggested line but may not be enough - let me know what you think:
> Consistent with the CNCF SIG definition, the mission of CNCF SIG Contributor Strategy is to collaborate on strategies related to building, scaling, and retaining contributor communities, including (people) governance, communications, operations, and tools. _We want to help grow flourishing, sustainable communities with smooth journeys throughout their CNCF project lifecycle._
To do that we will:

Reply:
You might know where I'm going with 'smooth journeys' but I don't think that would be good as final